### PR TITLE
About fragment

### PR DESCRIPTION
**Pull request by shertle**
Added about layouts and code.
Added blurbs for team and every team member
Added llama button

### CODE REVIEW

File: app/src/main/java/com/dgaf/happyhour/View/About.java [hunk 1]
```diff
@@ -2,28 +2,40 @@
 
 /**
  * Created by trentonrobison on 4/26/15.
+ * Written by Sherman Cheung
  */
 
+import android.os.AsyncTask;
```
Comment (t1miller) [inline app/src/main/java/com/dgaf/happyhour/View/About.java]: take out

**Comment by t1miller** [comment]
I did a line by line analysis of your code and have left notes on what needs to be dealt with. These are very important! Please update them in a timely manner.
Thanks,
Trent

**Comment by AdamDorwart** [comment]
Haha Trent the TA's might see these.

Also the images should be removed from the branch until they're actually implemented.

**Comment by brentonang** [comment]
Maybe keep the order consistent for xml properties like have id goes first, then height, then weight cause I see some where it switches height and weight. I'm just being picky, otherwise, great job!

File: app/src/main/res/layout/fragment_about.xml [hunk 2]
```diff
@@ -1,17 +1,198 @@
...
+
+            <ImageView
+                android:id="@+id/sherman_photo"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:src="@drawable/sherman_photo"/>
+
+            <ImageView
+                android:id="@+id/soham_photo"
+                android:layout_height="wrap_content"
```
Comment (brentonang) [inline app/src/main/res/layout/fragment_about.xml]: Switch height and width properties here to keep consistent with what we have

File: app/src/main/res/layout/fragment_about.xml [hunk 3]
```diff
@@ -1,17 +1,198 @@
...
+                android:layout_width="wrap_content"
+                android:src="@drawable/soham_photo"/>
+
+            <ImageView
+                android:id="@+id/trent_photo"
+                android:layout_height="wrap_content"
```
Comment (brentonang) [inline app/src/main/res/layout/fragment_about.xml]: Same here, width first then height

File: app/src/main/res/layout/fragment_about.xml [hunk 4]
```diff
@@ -1,17 +1,198 @@
...
         android:layout_height="wrap_content"
-        android:id="@+id/imageView"
-        android:layout_gravity="center"
-        android:src="@drawable/llama" />
+        android:orientation="vertical"
+        android:gravity="center_horizontal">
+
+        <Button
+            android:id="@+id/llamaButton"
+            android:layout_height="wrap_content"
```
Comment (brentonang) [inline app/src/main/res/layout/fragment_about.xml]: Here too, width then height

**Comment by brentonang** [comment]
I love my picture and my description. Thanks guys <3

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -16,7 +16,53 @@
...
+        the best value for food and drinks in their local area.\n\nOur team consists of Brenton
+        Ang, Sherman Cheung, Adam Dorwat, Xu(Ken) He, Soham Jatakia,
+        Chloe Lopez, David Liberman, Trent Miller, Anjali Patel, and Andrew Walters.  We are very
+        proud to bring you this app and wish you the best of luck in your gastronomic endeavors.
+        Happy Happy Hour!
+    </string>
+
+    <string name="about_us_title"><b>About Us</b></string>
+
+    <string name="about_us_david">Burrd is a fusion between blurred and bird. Why? Because BIRD
```
Comment (AdamDorwart) [inline app/src/main/res/values/strings.xml]: "Because _the_ BIRD is the word" -Gotta make sure we get that phrasing right ;)

**Comment by AdamDorwart** [comment]
I attempted merging this branch but there are way too many conflicts. Sherman, you're going to need to rebase your branch off of master. I can help you with this, this weekend.

**nvm I took care of it.

**Pull request merged**